.pydeps

```diff
@@ -0,0 +1,50 @@
+[pydeps]
+max_bacon = 32
+no_show = False
```

Comment:
should wecommit this? it can change anytime if the env file changes, so it'll get out of sync

Reply:
ahh Ibsee, ok!